Makefile

```
@@ -108,6 +112,60 @@ docker-build:
+	./bench --config config.yaml --pprof --output bench-results.json
+
+bench-go:
+	go test -bench=. -benchmem -count=3 -run='^$$' ./internal/percolator/ ./internal/scoring/
+
+# Cross-compile for Oracle ARM VM and run remotely.
+#   make vm-bench              — all phases
+#   make vm-bench BENCH_PHASES=percolator,scoring  — specific phases
+BENCH_PHASES ?= all
+BENCH_ARGS ?=
+
+vm-bench: vm-check-env
```

checkmake (0.3.2)

[warning] Target body for "vm-bench" exceeds allowed length of 5 lines (19).

(maxbodylength)